cppcache/integration/framework/config.h.in

```diff
@@ -21,5 +21,6 @@
 #define INTEGRATION_TEST_FRAMEWORK_CONFIG_H
 
 #define GFSH_EXECUTABLE "@Geode_gfsh_EXECUTABLE@"
+#define JAVAOBJECT_JAR_PATH "@JAVAOBJECT_JAR_PATH@"
```

Comment:
This is the project where the javaobject project should be dependent and variable defined.